homework/src/index.js

```diff
@@ -1,47 +1,119 @@
-'use strict';
-
+"use strict";
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status < 400) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
+  function main() {
+    fetchJSON("https://api.github.com/orgs/HackYourFuture/repos", (err, data) => {
+      function rend(changeEvent) {
```

Comment:
>A general comment: you have used cryptic variable and function names, such as rend which, I take it, stand for render. 
A general reply: Yes, it is render.
>Using cryptic names like that makes it hard for other people to understand your code (the JavaScript doesn't mind how you name things). You should write your code that best expresses your intentions in the clearest possible manner. 
In my opinion using large names can make a trouble for reading a code, code must be a poem, but a poem of code - is poem of logic, not the words.
>With moderns IDEs such VSCode there is really no benefit in typing effort to be gained from short names. Auto-complete will complete the name for you is you type the first few letters.
There is: large words creating a noise for an observer.
>You should as much as possible define functions at file level, rather than inside other functions. 
Before this comment, and our last lecture I never heard about it, and never got an explanation why. If its not difficult for you please explain.
>In the case of the rend() function I have a hard time to find out where it ends
Sure, I did this intentionally, because in the requirements wasn't written to make extra spaces for a better understanding.

Reply:
Hi Andrew, you are entitled to your own opinions. If you don't agree with my advice, so be it.